SchemaInventory.json

```diff
@@ -1173,7 +1173,7 @@
       "name": "ProcessPidGraphical",
       "path": "Domains\\4-Application\\Plant\\ProcessPidGraphical.ecschema.xml",
       "released": false,
-      "version": "01.00.02",
+      "version": "01.00.03",
```

Comment:
It will make the working copy and the latest released one having same versions. Working copy usually represents the next version e.g `01.00.04`.